src/amplitude-wrapper.js

```diff
@@ -155,14 +155,42 @@ var amplitudeUserAgentEnrichmentPlugin=function(i){"use strict";var e=function()
       const argsLength = args.length;
       const configuration = args[argsLength - 1];
 
-      if (configuration.defaultTracking.attribution) {
+      if (configuration.autocapture.attribution) {
         const excludeReferrers = [
-          ...(configuration.defaultTracking.attribution.excludeReferrersText || []),
-          ...(configuration.defaultTracking.attribution.excludeReferrersRegex?.map(item => new RegExp(item)) || [])
+          ...(configuration.autocapture.attribution.excludeReferrersText || []),
+          ...(configuration.autocapture.attribution.excludeReferrersRegex?.map(item => new RegExp(item)) || [])
         ];
-        delete configuration.defaultTracking.attribution.excludeReferrersText;
-        delete configuration.defaultTracking.attribution.excludeReferrersRegex;
-        configuration.defaultTracking.attribution.excludeReferrers = excludeReferrers;
+        delete configuration.autocapture.attribution.excludeReferrersText;
+        delete configuration.autocapture.attribution.excludeReferrersRegex;
+        // if excludeReferrers is empty, use default
+        if (excludeReferrers.length !== 0) {
```

Comment:
Is it still possible that customer provide empty excludeReferersText and excludeReferrersRegex to overwrite the default excludeReferrers?

Reply:
This is to fix the bug of the default behavior that when nothing is set, `excludeReferrers` is empty and then it override the default exclude referrers which is the top level domain. To fix it, we should not set `excludeReferrers` to an empty list. However, it's then not possible to set it as an empty list. But I think it's fine that, if customers want to include all referrer, they can just set a special value here.

Reply:
can we check real empty and [] and treat them differently?

Reply:
There is no difference between real empty and []